src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.MultiSegment.cs

```diff
@@ -2388,6 +2410,8 @@ private bool ConsumeSingleLineCommentMultiSegment(ReadOnlySpan<byte> localBuffer
                 idx = localBuffer.IndexOfAny(JsonConstants.LineFeed, JsonConstants.CarriageReturn);
                 if (idx != -1)
                 {
+                    toConsume = idx;
+                    end = new SequencePosition(_currentPosition.GetObject(), _currentPosition.GetInteger() + previousConsumed + leftOver + idx);
```

Comment:
`previousConsumed` is the function parameter and value passed to it is `_consumed` in the caller function `ConsumeCommentMultiSegment`. Looks like `previousConsumed` was confused with `prevTotalConsumed` a local variable which takes the value of `_totalConsumed`? In that case, I don't think we have an overflow here. Hope I am not missing something.

BTW, as you had pointed out in one of the previous comments, I guess the book keeping around `prevTotalConsumed` in this function is actually redundant because way up in the caller hierarchy that would be taken care of? Just asking. I didn't touch this part of code as this is something existing.